docker/api/Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /app
 COPY Gemfile* .ruby-version ./
 
 ARG BUNDLE_FLAGS
+RUN gem install bundler
```

Comment:
How was this working previously?
New Rails version changed something?

Reply:
I kept getting an error on bundler version mismatch etc etc. I think we were just stuck on the original bundler version as we weren't installing bundler each time?? Does bundler come pre-installed with ruby alpine?

Reply:
This has lead me down a bit of a rabbit hole as to the best way of keeping bundle up to date when using it in Dockerised apps.